app/plugins/Wallet/js/wallet.js

```diff
@@ -3,34 +3,17 @@
 const IPC = require('ipc');
 // Library for arbitrary precision in numbers
 const BigNumber = require('bignumber.js');
-// Variables to store api result values
+// Ensure precision
+BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ EXPONENTIAL_AT: 1e+9 });
```

Comment:
There's no way to make the UI's global variables available to the plugins. Since it's across a different index.html and functionally a different webpage being displayed, it has a different global scope than the UI's global scope.

Unless you mean it should be in the plugin's index.html